ReactSkia/components/RSkComponentTextInput.cpp

```diff
@@ -26,8 +29,12 @@ using namespace skia::textlayout;
 #define FONTSIZE_MULTIPLIER     10
 
 RSkComponentTextInput::RSkComponentTextInput(const ShadowView &shadowView)
-    : RSkComponent(shadowView)
-    ,isInEditingMode_(false){}
+    : RSkComponent(shadowView),
+    isInEditingMode_(false) {
+  RNS_LOG_DEBUG("called constructor");
+  eventCount_ = 0;
```

Comment:
we can initialize these variables similar to isInEditingMode_ 
( as its recommended way of initializing)